src/Portal/index.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * Portal
+ *
+ * Creates, and appends, a DOM node to the end of document.body and renders
+ * a React tree into it.
+ *
+ * Used with modals, dialogs, popovers.
+ */
+
+import * as React from 'react'
+import { createPortal } from 'react-dom'
+import PropTypes from 'prop-types'
+
+/**
+ * Forces a rerender
+ */
+function useForceUpdate() {
+  let [, dispatch] = React.useState(Object.create(null))
+  return React.useCallback(() => {
+    dispatch(Object.create(null))
+  }, [])
+}
+
+const Portal = ({ children, type = 'healthwise-ui-portal' }) => {
+  let mountNode = React.useRef(null)
```

Comment:
Doesn't really matter, but this could be a `const`, I believe?

Reply:
Could be a `const` but it could possibly be updated since it's a `ref`.

I know the current implementation isn't, but leaving it as a `let` should be fine.